src/python/crash_analysis/crash_comparer.py

```diff
@@ -53,9 +53,27 @@ def _similarity_ratio(string_1, string_2):
       1.0 * length_sum)
 
 
+def longest_common_subsequence(first_frames, second_frames):
+  """Count number of frames which are the same (taking into account order)."""
+  first_len = len(first_frames)
+  second_len = len(second_frames)
+
+  solution = [[0 for _ in range(second_len + 1)] for _ in range(first_len + 1)]
```

Comment:
shouldn't this be
[[0 for _ in range(second_len)] for _ in range(first_len)]
without the +1

Reply:
The solution array needs +1 in both dimentions for the extra [0][0] as the base case.